tests/test_cli.py

```diff
@@ -176,6 +180,17 @@ def test_command_add_objective(workdir, mocker):
     assert re.search(r'File ".*" is not a valid JSON file\.', res)
 
 
+def test_command_add_composite_traintuple_with_wrong_arguments(mocker, workdir):
+    json_file = workdir / "valid_json_file.json"
+    json_file.write_text(json.dumps({}))
+
+    res = client_execute(workdir, ['add', 'composite_traintuple', '--objective-key', 'foo',
+                                   '--algo-key', 'foo', '--dataset-key', 'foo',
+                                   '--data-samples-path', str(json_file), '--head-model-key', 'e'],
+                         exit_code=2)
+    assert re.search(r'The --trunk-model-key option is required when using --head-model-key\.', res)
```

Comment:
Could you also test it the other way around (with a trunk but no head) ? And also with both head and trunk to make sure they are accepted together?